internal/controller/components/workbenches/workbenches.go

```diff
@@ -51,19 +51,74 @@ func (s *componentHandler) NewCRObject(dsc *dscv1.DataScienceCluster) common.Pla
+		"odh-workbench-jupyter-minimal-cpu-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_CPU_PY311_IMAGE",
+		"odh-workbench-jupyter-minimal-cpu-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_CPU_PY312_IMAGE",
+		// Jupyter Workbench Images - Minimal CUDA
+		"odh-workbench-jupyter-minimal-cuda-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_CUDA_PY311_IMAGE",
+		"odh-workbench-jupyter-minimal-cuda-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_CUDA_PY312_IMAGE",
+		// Jupyter Workbench Images - Minimal ROCm
+		"odh-workbench-jupyter-minimal-rocm-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_ROCM_PY311_IMAGE",
+		"odh-workbench-jupyter-minimal-rocm-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_MINIMAL_ROCM_PY312_IMAGE",
+
+		// Jupyter Workbench Images - PyTorch CUDA
+		"odh-workbench-jupyter-pytorch-cuda-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_PYTORCH_CUDA_PY311_IMAGE",
+		"odh-workbench-jupyter-pytorch-cuda-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_PYTORCH_CUDA_PY312_IMAGE",
+		// Jupyter Workbench Images - PyTorch ROCm
+		"odh-workbench-jupyter-pytorch-rocm-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_PYTORCH_ROCM_PY311_IMAGE",
+		"odh-workbench-jupyter-pytorch-rocm-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_PYTORCH_ROCM_PY312_IMAGE",
+
+		// Jupyter Workbench Images - TensorFlow CUDA
+		"odh-workbench-jupyter-tensorflow-cuda-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_TENSORFLOW_CUDA_PY311_IMAGE",
+		"odh-workbench-jupyter-tensorflow-cuda-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_TENSORFLOW_CUDA_PY312_IMAGE",
+		// Jupyter Workbench Images - TensorFlow ROCm
+		"odh-workbench-jupyter-tensorflow-rocm-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_TENSORFLOW_ROCM_PY311_IMAGE",
+
+		// Jupyter Workbench Images - TrustyAI CPU
+		"odh-workbench-jupyter-trustyai-cpu-py311-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_TRUSTYAI_CPU_PY311_IMAGE",
+		"odh-workbench-jupyter-trustyai-cpu-py312-ubi9-n": "RELATED_IMAGE_ODH_WORKBENCH_JUPYTER_TRUSTYAI_CPU_PY312_IMAGE",
+
+		// Pipeline Runtime Images
+		"odh-pipeline-runtime-datascience-cpu-py311-ubi9-n": "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_DATASCIENCE_CPU_PY311_IMAGE",
+		"odh-pipeline-runtime-datascience-cpu-py312-ubi9-n": "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_DATASCIENCE_CPU_PY312_IMAGE",
+		"odh-pipeline-runtime-minimal-cpu-py311-ubi9-n":     "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_MINIMAL_CPU_PY311_IMAGE",
+		"odh-pipeline-runtime-minimal-cpu-py312-ubi9-n":     "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_MINIMAL_CPU_PY312_IMAGE",
+		"odh-pipeline-runtime-tensorflow-cuda-py311":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_CUDA_PY311_IMAGE",
+		"odh-pipeline-runtime-tensorflow-cuda-py312":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_CUDA_PY312_IMAGE",
+		"odh-pipeline-runtime-tensorflow-rocm-py311":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_ROCM_PY311_IMAGE",
```

Comment:
```suggestion
		"odh-pipeline-runtime-tensorflow-cuda-py311-ubi9-n":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_CUDA_PY311_IMAGE",
		"odh-pipeline-runtime-tensorflow-cuda-py312-ubi9-n":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_CUDA_PY312_IMAGE",
		"odh-pipeline-runtime-tensorflow-rocm-py311-ubi9-n":        "RELATED_IMAGE_ODH_PIPELINE_RUNTIME_TENSORFLOW_ROCM_PY311_IMAGE",
```

^to match current state of keys in params-latest.env on main: https://github.com/red-hat-data-services/notebooks/blob/main/manifests/base/params-latest.env

(all other keys appear to be correct, as do all of the vars used as values, based on what's in the issue description in Jira)